docs/hyperspectral_rot90.md

```diff
@@ -0,0 +1,43 @@
+## Rotate Hyperspectral Datacubes 
+
+This function rotates a hyperspectral datacube counterclockwise in increments of 90 degrees.  The input and output is a [`Spectral_data` class](Spectral_data.md) 
+instance created while reading in with [`pcv.readimage`](read_image.md) with `mode='envi'`. This function is similar to the [`pcv.rotate`](rotate2.md) functions
+but is specifically suitable for HSI image analysis. 
+
+**plantcv.hyperspectral.rot90**(*spectral_data, k*)
+
+**returns** rot_hsi (instance of the `Spectral_data` class)
+
+- **Parameters:**
+    - spectral_data      - Hyperspectral data instance
+    - k                  - Number of times the array is rotated by 90 degrees
+
+
+- **Example use:**
+    - Below
+    
+    
+**Input image**
+
+![Screenshot](img/documentation_images/rotate2/34_whitebalance.jpg)
+
+```python
+
+from plantcv import plantcv as pcv
+
+# Set global debug behavior to None (default), "print" (to file), 
+# or "plot" (Jupyter Notebooks or X11)
+pcv.params.debug = "plot"
+
+# Many plants absorb 430nm wavelength light. 
+rot_hsi = pcv.hyperspectral.rot90(spectral_data=spectral_array_obj, k=1)
+
+```
+
+
+**Image after rotating 90 degrees (k=1)**
+
+![Screenshot](img/documentation_images/rotate2/10_rotated_img.jpg)
```

Comment:
This image is rotated 10 degrees. It looks like it's the same images that are on the `plantcv.transform.rotate` doc page, where it gets rotated 10 degrees

Reply:
Oh no, good catch. I believe I copy and pasted that doc page to help myself with formatting but forgot to update that line 😅 I'll dig up the actual img I meant to link.

Reply:
Jk, looks like I never created an example image. We can just delete the example images from this doc page. None of our other HSI docs have example images either.